target/linux/ramips/image/mt7621.mk

```diff
@@ -203,6 +203,36 @@ define Device/rb750gr3
 endef
 TARGET_DEVICES += rb750gr3
 
+define Device/rbm33g
+  DTS := RBM33G
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := MikroTik RBM33G
+  BOARDNAME := MIKROTIK-RBM33G
+  DEVICE_PACKAGES := kmod-usb3 uboot-envtools
```

Comment:
Please keep in mind, that these packages will not be included in the initramfs build by the buildbot. It is a limitation of the CONFIG_TARGET_PER_DEVICE_ROOTFS which is set by the buildbots.

Reply:
Thanks, I was not aware of that.